resources/zib/zib-SNAQ65+Score.xml

```diff
@@ -0,0 +1,257 @@
+      </slicing>
+    </element>
+    <element id="Observation.effective[x]:effectiveDateTime">
+      <path value="Observation.effective[x]"/>
+      <sliceName value="effectiveDateTime"/>
+      <short value="SNAQScoreDateTime"/>
+      <definition value="The date and time at which the SNAQ65+ score was determined."/>
+      <alias value="SNAQ65+ScoreDatumTijd"/>
+      <type>
+        <code value="dateTime"/>
+      </type>
+      <mapping>
+        <identity value="zib-snaq65plusscore-v1.2-2020EN"/>
+        <map value="NL-CM:4.30.6"/>
+        <comment value="SNAQScoreDateTime"/>
+      </mapping>
+    </element>
+    <element id="Observation.value[x]">
+      <path value="Observation.value[x]"/>
+      <slicing>
+        <discriminator>
+          <type value="type"/>
+          <path value="$this"/>
+        </discriminator>
+        <rules value="open"/>
+      </slicing>
+    </element>
+    <element id="Observation.value[x]:valueInteger">
+      <path value="Observation.value[x]"/>
+      <sliceName value="valueInteger"/>
+      <short value="TotalScore"/>
+      <definition value="The total score is the sum of all the individual scores. The total score has a range from 0 - 5. The relation between the value of the total score and the color outcome of the score tool is:&lt;br /&gt;0-1 = green&lt;br /&gt;2= orange&lt;br /&gt;3 and higher = red."/>
+      <alias value="TotaalScore"/>
+      <type>
+        <code value="integer"/>
+      </type>
+      <minValueInteger value="0"/>
+      <maxValueInteger value="5"/>
+      <mapping>
+        <identity value="zib-snaq65plusscore-v1.2-2020EN"/>
+        <map value="NL-CM:4.30.2"/>
+        <comment value="TotalScore"/>
+      </mapping>
+    </element>
+    <element id="Observation.note">
+      <path value="Observation.note"/>
+      <max value="1"/>
+    </element>
+    <element id="Observation.note.text">
+      <path value="Observation.note.text"/>
+      <short value="Comment"/>
+      <definition value="A comment on the circumstances during the measurement."/>
+      <alias value="Toelichting"/>
+      <mapping>
+        <identity value="zib-snaq65plusscore-v1.2-2020EN"/>
+        <map value="NL-CM:4.30.7"/>
+        <comment value="Comment"/>
+      </mapping>
+    </element>
+    <element id="Observation.component">
+      <path value="Observation.component"/>
+      <slicing>
+        <discriminator>
+          <type value="value"/>
+          <path value="code"/>
+        </discriminator>
+        <rules value="open"/>
+      </slicing>
+    </element>
+    <element id="Observation.component:weightLossScore">
+      <path value="Observation.component"/>
+      <sliceName value="weightLossScore"/>
+      <max value="1"/>
+    </element>
+    <element id="Observation.component:weightLossScore.code">
+      <path value="Observation.component.code"/>
+      <patternCodeableConcept>
+        <coding>
+          <system value="2.16.840.1.113883.2.4.3.11.60.40.4.0.1"/>
```

Comment:
Shouldn't the value be "urn:oid:2.16.840.1.113883.2.4.3.11.60.40.4.0.1"? Also applies to other scores.